x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -99,6 +108,9 @@ export default function (ctx: FtrProviderContext) {
       });
 
       after(async () => {
+        if (await PageObjects.share.isShareModalOpen()) {
+          await PageObjects.share.closeShareModal();
+        }
```

Comment:
In general I'm not a huge fan of sticking this check into every test - is this an artifact of other tests not properly cleaning themselves up?